### PR TITLE
Blogpost companion code: schedule data wrangler flow with AWS Lambda function

### PR DESCRIPTION
**Pull request by chenliu0831**
*Issue #, if available:* N/A

*Description of changes:*
This is the companion code that will be a companion to an upcoming blog post. 

*Testing done:* 

Manually follow the blog post. There are no notebooks. 

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [ ] I have tested my notebook(s) and ensured it runs end-to-end
- [ ] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    print(f"ProcessingInput defined")
+
+    # IAM role for executing the processing job. You can set this up with your own role
+    iam_role = "<IAM-ROLE-PLACEHOLDER>"
+
+    # Unique processing job name. Please give a unique name every time you re-execute processing jobs
+    processing_job_name = f"data-wrangler-flow-processing-{flow_export_id}"
+
+    # Data Wrangler Container URL.
+    container_uri = "663277389841.dkr.ecr.us-east-1.amazonaws.com/sagemaker-data-wrangler-container:1.x"
```
Comment (aaronmarkham) [inline sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py:137]: Is this only available in us-east-1?

File: sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        input_name="flow",
+        s3_data_type="S3Prefix",
+        s3_input_mode="File",
+        s3_data_distribution_type="FullyReplicated"
+    )
+
+    print(f"ProcessingInput defined")
+
+    # IAM role for executing the processing job. You can set this up with your own role
+    iam_role = "<IAM-ROLE-PLACEHOLDER>"
```
Comment (aaronmarkham) [inline sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py:131]: Kind of buried and a required edit by the customer.... can this be tied to sagemaker session instead and assume they've setup their roles... and then this script can be tested?

File: sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    # bucket = <my-own-storage-bucket>
+    bucket = sess.default_bucket()
+    print(f"Data Wrangler export storage bucket: {bucket}")
+
+    # unique flow export ID
+    flow_export_id = f"{time.strftime('%Y-%m-%d-%H-%M-%S', time.gmtime())}-{str(uuid.uuid4())[:8]}"
+    flow_export_name = f"flow-{flow_export_id}"
+
+    # Output name is auto-generated from the select node's ID + output name from the flow file.
+    output_name = "8b392709-d2c4-4b8e-bdda-e75b2d14f35e.default"
```
Comment (aaronmarkham) [inline sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py:91]: Won't this break for the customer if they try to run this?

**Review by aaronmarkham — COMMENTED** [review]
Can you make sure to follow up with a link to the blog in the comments and/or add a readme to this folder?

Also, some of the hard-coding seems like this won't be easy or possible to test.

**Comment by chenliu0831** [comment]
Moved to AWS samples instead. Closing

**Pull request closed without merging**